vendor/sigs.k8s.io/cloud-provider-azure/pkg/azclient/arm_conf.go

```diff
@@ -49,6 +49,8 @@ type ARMClientConfig struct {
 	// when setting AzureAuthConfig.Cloud with "AZURESTACKCLOUD" to customize ARM endpoints
 	// while the cluster is not running on AzureStack.
 	DisableAzureStackCloud bool `json:"disableAzureStackCloud,omitempty" yaml:"disableAzureStackCloud,omitempty"`
+	// If true, HTTP responses' retry-after header will be overridden with the configured minimum retry-after value if lower than the configured minimum
+	EnableRetryAfterOverride bool `json:"enableRetryAfterOverride,omitempty" yaml:"enableRetryAfterOverride,omitempty"`
```

Comment:
could you make changes in https://github.com/kubernetes-sigs/cloud-provider-azure repo first? thanks